modules/local/mcl_test_samplesheet_check.nf

```diff
@@ -0,0 +1,57 @@
+process MCL_TEST_SAMPLESHEET_CHECK {
```

Comment:
Why do we need a separate process? Is it different from what the other `samplesheet_check` module does?

Reply:
Nope - deleted it.